docs/en/writing-running-appium/tutorial/swipe/simple-element.md

```diff
@@ -0,0 +1,206 @@
+## Simple Element swipe
+
+### Android: UIAutomator2
+
+With UIAutomator2 driver, by default, we see only elements on screen. If
```

Comment:
By default UIAutomator2 driver only returns element, which are visible on the screen. If a container is split between the visible and invisible parts of the viewport then a part of its content won't be visible